app/build.gradle

```diff
@@ -1,7 +1,7 @@
-plugins {
-    id 'com.android.application'
-    id 'kotlin-android'
-}
+apply plugin: 'com.android.application'
```

Comment:
Can't we still use the `plugins` structure? I believe the `apply plugin: ...` syntax is going to be deprecated soon

Reply:
Changed